packages/package_info_plus/package_info_plus/example/ios/Flutter/Flutter.podspec

```diff
@@ -1,18 +1,18 @@
 #
 # NOTE: This podspec is NOT to be published. It is only used as a local source!
+#       This is a generated file; do not edit or check into version control.
 #
 
 Pod::Spec.new do |s|
   s.name             = 'Flutter'
   s.version          = '1.0.0'
   s.summary          = 'High-performance, high-fidelity mobile apps.'
-  s.description      = <<-DESC
-Flutter provides an easy and productive way to build and deploy high-performance mobile apps for Android and iOS.
-                       DESC
-  s.homepage         = 'https://plus.fluttercommunity.dev'
+  s.homepage         = 'https://flutter.io'
   s.license          = { :type => 'MIT' }
-  s.author           = { 'Flutter Community' => 'authors@fluttercommunity.dev' }
+  s.author           = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com' }
   s.source           = { :git => 'https://github.com/flutter/engine', :tag => s.version.to_s }
   s.ios.deployment_target = '8.0'
-  s.vendored_frameworks = 'Flutter.framework'
```

Comment:
We want to keep this as is.

Reply:
Sorry must be a merge conflict. I did not mean to edit this one.